### PR TITLE
Add wbg

### PR DESCRIPTION
**Pull request by SrEstegosaurio**
Add [wbg](https://codeberg.org/dnkl/wbg) to the list of derivations.

wbg is a small utility to set a wallpaper image written in C.

P.S: I'm very new to contributing so I'm sorry if I done something dumb or just wasted your time. Let me know if that's the case.

### CODE REVIEW

**Comment by colemickens** [comment]
Did you maybe re-format the tree? I see some other edits in there. Not a huge deal, just curious.

File: flake.nix [hunk 1]
```diff
@@ -153,6 +153,13 @@
               extra.buildInputs = [ prev.libdrm ];
               replace.doCheck = false;
             }
+            {
+              attrName = "wbg";
+              extra.buildInputs = [ prev.pixman ];
+              replaceInput = {
+                wayland-protocols = final.new-wayland-protocols;
```
Comment (colemickens) [inline flake.nix:160]: does it specifically need the newer one?

**Comment by SrEstegosaurio** [comment]
I did ran a `nix fmt`. It seemed like the correct thing to do.
I thought it only affected my code. 🤔

File: flake.nix (hunk 1 above)
Comment (SrEstegosaurio) [reply flake.nix:160]: I don't know? Probably not, I spent a few minutes observing how things were done and  thought that if the project had dependencies that can be provided by `nixpkgs-wayland` they should.

Although  I have not run any tests.

**Comment by Artturin** [comment]
The formatting changes can stay, we should make the CI run `nix fmt` automatically.

File: flake.nix (hunk 1 above)
Comment (Artturin) [reply flake.nix:160]: > I don't know. Probably not. I've spent a few minutes observing how things were done and thought that if the project had dependencies that can be provided by `nixpkgs-wayland` they should.
> 
> Although I have not run any tests.

If it builds with the wayland-protocols from nixpkgs then the replacement isn't needed

**Comment by Artturin** [comment]
Committed in https://github.com/nix-community/nixpkgs-wayland/commit/edba7d689bc9559398d71fa531c052074bc86ef3

Had to do it that way instead of editing your PR as maintainers cannot push to the PR's branch when master is used

I recommend reading https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#how-to-create-pull-requests

**Comment by SrEstegosaurio** [comment]
Sorry for not having read it beforehand. Thanks for pointing me to the documentation.

**Pull request closed without merging**